### DISCUSSION TITLE
access desktop account on mobile

### DISCUSSION THREAD

**Discussion by bwallx** [mastodon/mastodon#19677] [Q&A] [answered]
I have an account on my PC and I am trying to set up the android app on my phone.

It is asking for a server. I can not remember which server I signed up on with my desktop. I tried a couple on the phone, but it doesn't recognise my username/password from my desktop.

**Comment by Gargron** [comment, marked as answer]
You can see the server on desktop in the browser's address bar, for example https://**mastodon.social**/example
